### DISCUSSION TITLE
got no response when execute 'which run_metplus.py'

### DISCUSSION THREAD

**Discussion by rkong66** [dtcenter/METplus#1298] [Configuration] [answered]
Hello,
    I am using AWS cloud for METplus training. The script run_metplus.py that used to work last week does not work right now. When I execute 'which run_metplus.py'. I got no response. Any idea for that?

(base) ec2-user@ip-172-31-91-120:/d1/METplus-4.0.0_Tutorial$ source METplus-4.0.0_TutorialSetup.sh
(base) ec2-user@ip-172-31-91-120:/d1/METplus-4.0.0_Tutorial$ which run_metplus.py
(base) ec2-user@ip-172-31-91-120:/d1/METplus-4.0.0_Tutorial$

Best
Rong

**Comment by hankenstein2** [comment]
That is odd, it looks like your METplus-4.0.0_TutorialSetup.sh got reset
somehow. This is a great opportunity to work on the setup.

Can you find the following lines in your METplus-4.0.0_TutorialSetup.sh and
make them look like below and then source that file again and let me know
how it goes.

Hank

# Directory to store tutorial files - configurations, output, notes, etc.
export METPLUS_TUTORIAL_DIR=/d1//METplus-4.0.0_Tutorial

# Path to the METplus installation location
export METPLUS_BUILD_BASE=/opt/METplus

# Path to the MET installation location
export MET_BUILD_BASE=/opt/MET

# Path to input data required for running use cases
export METPLUS_DATA=/d1/projects/METplus/METplus_Data





On Tue, Dec 7, 2021 at 10:12 AM Rong Kong ***@***.***> wrote:

> Hello,
> I am using AWS cloud for METplus training. The script run_metplus.py that
> used to work last week does not work right now. When I execute 'which
> run_metplus.py'. I got no response. Any idea for that?
>
> (base) ***@***.***:/d1/METplus-4.0.0_Tutorial$ source
> METplus-4.0.0_TutorialSetup.sh
> (base) ***@***.***:/d1/METplus-4.0.0_Tutorial$ which
> run_metplus.py
> (base) ***@***.***:/d1/METplus-4.0.0_Tutorial$
>
> Best
> Rong
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus/discussions/1298>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHAUIKEK6KNV33VOQY5XIELUPY56LANCNFSM5JRYSERQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
--------------------------------------------------------------
Hank Fisher       Research Applications Laboratory
303-497-2817      National Center for Atmospheric Research
***@***.***  P.O. Box 3000, Boulder CO 80307-3000
--------------------------------------------------------------

**Reply by rkong66** [reply]
Thank you for the reply. I tried to source the new script, but still got no response.   

(base) ec2-user@ip-172-31-91-120:/d1/METplus-4.0.0_Tutorial$ source METplus-4.0.0_TutorialSetup.sh
(base) ec2-user@ip-172-31-91-120:/d1/METplus-4.0.0_Tutorial$ which run_metplus.py
(base) ec2-user@ip-172-31-91-120:/d1/METplus-4.0.0_Tutorial$ module list
-bash: module: command not found
(base) ec2-user@ip-172-31-91-120:/d1/METplus-4.0.0_Tutorial$


Rong

**Comment by hankenstein2** [comment]
That is odd. I think I logged into your ip 52.201.226.251 as admin and
sourced that file and got a correct result.

Can you tell me the results of the following command...

printenv METPLUS_BUILD_BASE

Hank

On Tue, Dec 7, 2021 at 10:34 AM Rong Kong ***@***.***> wrote:

> Thank you for the reply. I tried to source the new script, but still got
> no response.
>
> (base) ***@***.***:/d1/METplus-4.0.0_Tutorial$ source
> METplus-4.0.0_TutorialSetup.sh
> (base) ***@***.***:/d1/METplus-4.0.0_Tutorial$ which
> run_metplus.py
> (base) ***@***.***:/d1/METplus-4.0.0_Tutorial$ module list
> -bash: module: command not found
> (base) ***@***.***:/d1/METplus-4.0.0_Tutorial$
>
> Rong
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus/discussions/1298#discussioncomment-1767664>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHAUIKGA7NUXFHQ53KWMC5LUPZATVANCNFSM5JRYSERQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
--------------------------------------------------------------
Hank Fisher       Research Applications Laboratory
303-497-2817      National Center for Atmospheric Research
***@***.***  P.O. Box 3000, Boulder CO 80307-3000
--------------------------------------------------------------

**Comment by rkong66** [comment]
The answer is: 
(base) ec2-user@ip-172-31-91-120:/d1/METplus-4.0.0_Tutorial$ printenv METPLUS_BUILD_BASE
/opt/METplus
Rong

**Reply by rkong66** [reply]
You can login into my ip 52.201.226.251 as a common user, I did not change the password.

**Comment by hankenstein2** [comment, marked as answer]
Ah, I think I see the issue. You still need this entry in your .sh file

# Add METplus ush directory and MET bin directory to the path
# This allows you to run METplus without specifying the full path
export PATH=${METPLUS_BUILD_BASE}/ush:${MET_BUILD_BASE}/bin:${PATH}

Hank



On Tue, Dec 7, 2021 at 10:44 AM Rong Kong ***@***.***> wrote:

> The answer is:
> (base) ***@***.***:/d1/METplus-4.0.0_Tutorial$ printenv
> METPLUS_BUILD_BASE
> /opt/METplus
> Rong
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus/discussions/1298#discussioncomment-1767722>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHAUIKBRJDZPPEWXBQPUB3LUPZBXZANCNFSM5JRYSERQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
--------------------------------------------------------------
Hank Fisher       Research Applications Laboratory
303-497-2817      National Center for Atmospheric Research
***@***.***  P.O. Box 3000, Boulder CO 80307-3000
--------------------------------------------------------------

**Reply by rkong66** [reply]
Yes, it works right now. Thank you for your help.

Best
Rong

**Comment by hankenstein2** [comment]
Good to hear!

Hank

On Tue, Dec 7, 2021 at 10:49 AM Rong Kong ***@***.***> wrote:

> Yes, it works right now. Thank you for your help.
>
> Best
> Rong
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus/discussions/1298#discussioncomment-1767766>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHAUIKBYSG7IXO6WNSEODFLUPZCMNANCNFSM5JRYSERQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
--------------------------------------------------------------
Hank Fisher       Research Applications Laboratory
303-497-2817      National Center for Atmospheric Research
***@***.***  P.O. Box 3000, Boulder CO 80307-3000
--------------------------------------------------------------
